tests/test_config_value.py

```diff
@@ -1700,6 +1700,12 @@ def test_eval():
         'bad4' : { 'type' : 'Eval', 'str' : 'np.exp(-0.5 * q**2)', 'fx' : 1.8 },
         'bad5' : { 'type' : 'Eval', 'eval_str' : 'np.exp(-0.5 * x**2)', 'fx' : 1.8 },
 
+        # Check that a list can be made using Eval
+        'list0' : [0,1,2,3,4,5],
+        'list1' : '$np.arange(6)',
+        'list2' : (0,1,2,3,4,5),
+        'list3' : '$(0,1,2,3,4,5)',
+
```

Comment:
Looks good.  Do you want to test `Eval`-ing a LookupTable and dict too, since you just added those?

Reply:
Thanks.  This was a good idea.  These were fine as implemented, but it turned up other places where some config code assumed that all dicts would have str keys, which isn't necessarily the case for objects that are intended to be a simple dict, rather than a construction directive.